apps/assisted-disconnected-ui/src/components/ResetSingleClusterModal.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {
+  Button,
+  Modal,
+  ModalVariant,
+  ButtonVariant,
+  Text,
+  TextContent,
+} from '@patternfly/react-core';
+import { useModalDialogsContext, ClustersService } from '@openshift-assisted/ui-lib/ocm';
+import { getApiErrorMessage, handleApiError, ErrorState } from '@openshift-assisted/ui-lib/common';
+import { useNavigate } from 'react-router-dom-v5-compat';
+
+const ResetSingleClusterModal: React.FC = () => {
+  const navigate = useNavigate();
+  const [error, setError] = React.useState<{ title: string; message: string } | null>(null);
+  const { resetSingleClusterDialog } = useModalDialogsContext();
+  const { data, isOpen, close: onClose } = resetSingleClusterDialog;
+  const cluster = data?.cluster;
+
+  if (!cluster) {
+    return null;
+  }
+
+  const handleClose = () => {
+    setError(null);
+    onClose();
+  };
+
+  const handleResetAsync = async () => {
+    try {
+      await ClustersService.remove(cluster.id);
+      navigate(`/`);
+    } catch (e) {
+      handleApiError(e, () => {
+        setError({
+          title: 'Failed to reset cluster installation',
+          message: getApiErrorMessage(e),
+        });
+      });
+    }
+  };
+
+  const getModalContent = () => {
+    if (error) {
+      return <ErrorState title={error.title} content={error.message} />;
+    }
+
+    return (
+      <TextContent>
+        <Text component="p">
+          This will reset the installation and return to the cluster configuration. Some hosts may
+          need to be re-registered by rebooting into the Discovery ISO.
+        </Text>
+
+        <Text component="p">Are you sure you want to reset the cluster?</Text>
+      </TextContent>
+    );
+  };
+
+  const actions = [
+    <Button key="reset" variant={ButtonVariant.danger} onClick={() => void handleResetAsync()}>
+      Reset Cluster
+    </Button>,
+  ];
+
+  actions.push(
+    <Button key="cancel" variant={ButtonVariant.link} onClick={handleClose}>
+      Cancel
+    </Button>,
+  );
```

Comment:
why so complicated ?
We should also i18n the strings
```suggestion
  const actions = [
    <Button key="reset" variant={ButtonVariant.danger} onClick={() => void handleResetAsync()}>
      {t("ai:Reset Cluster")}
    </Button>,
    <Button key="cancel" variant={ButtonVariant.link} onClick={handleClose}>
       {t("ai:Reset Cancel")}
    </Button>
  ];
```

the bnts should also be disabled while loading